src/main/java/pokecube/legends/init/BlockInit.java

```diff
@@ -328,11 +347,33 @@ public static void init()
 
         for (final RegistryObject<Block> reg : PokecubeLegends.BLOCKS_TAB.getEntries())
         {
-            // These are registered seperately, so skip them.
+            // These are registered separately, so skip them.
             if (reg == BlockInit.ULTRA_TORCH1 || reg == BlockInit.ULTRA_TORCH1_WALL) continue;
             PokecubeLegends.ITEMS.register(reg.getId().getPath(), () -> new BlockItem(reg.get(), new Item.Properties()
                     .group(PokecubeLegends.TAB)));
         }
     }
 
+    public static class PokecubeWoodStairs extends StairsBlock 
+    {
+        @SuppressWarnings("deprecation")
+		public PokecubeWoodStairs() {
+            super(Blocks.OAK_PLANKS.getDefaultState(), Block.Properties.from(Blocks.OAK_PLANKS)
+                    .sound(SoundType.WOOD)
+                    .hardnessAndResistance(2.0f, 3.0f)
+            );
+        }
+    }
+    
+//	public static void addStrippable(final RegistryObject<Block> log, RegistryObject<Block> strippedLog) {
```

Comment:
This is the code block that isn't functional for stripping logs. The `.put` doesn't take the `RegistryObject<Block>`, any suggestions to fix this?